### PR TITLE
Ingest mtsummit 1989

### PR DESCRIPTION
**Pull request by davidstap**
See https://aclanthology.org/www.mt-archive.info/70/MTS-1989-TOC.htm for original. (I accidentally started with the 2nd edition of the conference, I'll add the first one next.)

* The publisher is unknown so I left this field empty.
* I did not include the Opening Session & Closing Address. I could maybe add the Opening Session as frontmatter, but not sure at which volume. What do you think?
* Not sure what to do with "Exhibitors".

### CODE REVIEW

**Comment by mjpost** [comment]
Don't forget to pull the latest `master` before branching. I just merged it in.

Sorry of this is a pain, but I think this is "overvolumed". I think we should just process this entire conference as a single volume with a name of `1`. It is too much of a mess to make sense of and there doesn't seem to have been a formal proceedings issued, at least not that we have here (e.g., the opening address is page 155).

**Comment by mjpost** [comment]
Also it looks like one of the authors needs to be disambiguated:

image

There is code that does this on ingest in `ingest.py` that you might want to copy, or just handle this manually, you'll probably be okay.

**Comment by davidstap** [comment]
Okay, I'll process as a single volume ASAP. I'll do the same for other MT Summit editions that look similar.

**Comment by davidstap** [comment]
I have processed this now as a single volume.

File: data/xml/1989.mtsummit.xml [hunk 1]
```diff
@@ -0,0 +1,180 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="1989.mtsummit">
+  <volume id="1" ingest-date="2021-06-28">
+    <meta>
+      <booktitle>Proceedings of Machine Translation Summit II</booktitle>
+      <publisher>?? Not mentionned on TOC</publisher>
```
Comment (mjpost) [inline data/xml/1989.mtsummit.xml:6]: delete line

File: data/xml/1989.mtsummit.xml [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+      <author><first>Sergei</first><last>Perschke</last></author>
+      <url hash="253c1f08">1989.mtsummit-1.14</url>
+    </paper>
+    <paper id="15">
+      <title><fixed-case>J</fixed-case>apanese view of the future of machine translation</title>
+      <author><first>Makoto</first><last>Nagao</last></author>
+      <url hash="40b4eadf">1989.mtsummit-1.15</url>
+    </paper>
+    <paper id="16">
+      <title></title>
```
Comment (mjpost) [inline data/xml/1989.mtsummit.xml:92]: titles missing here and in many other places

**Review by mjpost — COMMENTED** [review]
See notes. There seem to be two other missing papers:

1. The J. Gachot link under Session 1
2. The Nagao special lecture

**Comment by davidstap** [comment]
> See notes. There seem to be two other missing papers:
> 
> 1. The J. Gachot link under Session 1
> 2. The Nagao special lecture

The J. Gachot is paper 3 ("The SYSTRAN renaissance", this is the actual title if you open the pdf.)
The Nagao special lecture (Japanese view of the future of machine translation) is paper 15. (I grouped them by their type, should I revert this or is it okay?)

**Comment by mjpost** [comment]
Oh, I see. I think it makes most sense to follow the program order here, since we are just sticking with a single volume for this one.

File: data/xml/1989.mtsummit.xml [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+      <author><first>Mahmoud Esma’il</first><last>Sieny</last></author>
+      <url hash="93721624">1989.mtsummit-1.24</url>
+    </paper>
+    <paper id="25">
+      <title>Computational Linguistics and <fixed-case>MT</fixed-case> in <fixed-case>I</fixed-case>taly</title>
+      <author><first>Antonio</first><last>Zampolli</last></author>
+      <url hash="e0014765">1989.mtsummit-1.25</url>
+    </paper>
+    <paper id="26">
+      <title>Statements by mr. <fixed-case>C</fixed-case>.<fixed-case>N</fixed-case>. <fixed-case>J</fixed-case>ansen van <fixed-case>R</fixed-case>osendaal</title>
```
Comment (mjpost) [inline data/xml/1989.mtsummit.xml:139]: Mr. should probably be capitalized

**Review by mjpost — APPROVED** [review]
LGTM after the fix.

**Pull request merged**